openslides/core/projector.py

```diff
@@ -139,3 +139,10 @@ class Message(ProjectorElement):
     def check_data(self):
         if self.config_entry.get('message') is None:
             raise ProjectorException('No message given.')
+
+
+class ListOfSpeakersOverlay(ProjectorElement):
+    """
+    List of speakers (last, current, next) on the projector. Rendered as overlay.
+    """
+    name = 'core/speakeroverlay'
```

Comment:
Hm. I think the `get_requirements()` is missing. Please test to view the projector with a user which has only the permission to see the projector, nothing else (no permission to see agenda, motions, users, ...). Then the access to all these objects should be restricted but not if the element is on the projector. See utils/projector.py for more info.